toolchains/arm32.cmake

```diff
@@ -17,6 +17,7 @@ set(CMAKE_C_COMPILER /usr/bin/arm-linux-gnueabihf-gcc CACHE PATH "" FORCE)
 set(CMAKE_CXX_COMPILER /usr/bin/arm-linux-gnueabihf-g++ CACHE PATH "" FORCE)
 
 set(TOIT_SYSTEM_NAME ${CMAKE_SYSTEM_NAME})
+unset(TOIT_BUILD_BOOT_SNAPSHOT)
```

Comment:
This actually doesn't work, as subdirectories can't unset super variables.
Didn't find a good way to do it, so I changed it to do the opposite: the subdirectories set a `TOIT_IS_CROSS` variable which is then used to avoid building the snapshot. https://github.com/dsobotta/toit/pull/12

Reply:
Thanks for the thorough feedback. I'll get started on the changes now. Very interesting that it was failing on win64; I haven't actually tried that target yet. The changes you PR'd look like a good approach regardless.

Reply:
It wasn't actually failing.
The `if (defined` wasn't actually triggered and the dependency was always added.
But since the Makefile used just `ninja` that wasn't a problem as the `build_toitvm` rule wasn't invoked.

Reply:
merged